README.md

```diff
@@ -43,10 +74,15 @@ navigation menu:
 [lens]: https://github.com/lensapp/lens
 [starboard]: https://github.com/aquasecurity/starboard
 [starboard-crds]: https://github.com/aquasecurity/starboard#custom-security-resources-definitions
+[starboard-getting-started]: https://github.com/aquasecurity/starboard#getting-started
 
 [lens-installation]: https://github.com/lensapp/lens#installation
 [lens-releases]: https://github.com/lensapp/lens/releases
+[release-img]: https://img.shields.io/github/release/aquasecurity/starboard-lens-extension.svg?logo=github
```

Comment:
I didn't know you could define anchors/variables like this in markdown. Does this work for any markdown flavour or just Github?

Reply:
According to https://www.markdownguide.org/basic-syntax/#reference-style-links it's not just GitHub flavour